.github/workflows/primeqa-ci.yml

```diff
@@ -42,9 +42,9 @@ jobs:
           echo "Run IR tests"
           pytest -s --cov primeqa/ir --cov-config .coveragerc --cov-fail-under=50 tests/primeqa/ir
           echo "Run MRC tests excluding trainer"
-          pytest -s --cov primeqa/mrc/processors --cov-config .coveragerc --cov-fail-under=50 --ignore tests/primeqa/mrc/trainers/test_mrc.py  tests/primeqa/mrc
```

Comment:
Why did this need to be changed?

Reply:
Because the CI was not running the new unit test but was reporting the coverage was low.